WordPress/Classes/Services/MenusService.h

```diff
@@ -0,0 +1,103 @@
+#import "LocalCoreDataService.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class Blog;
+@class Menu;
+@class MenuLocation;
+@class MenuItem;
+
+typedef void(^MenusServiceSuccessBlock)();
+typedef void(^MenusServiceCreateMenuRequestSuccessBlock)(NSNumber *menuID);
+typedef void(^MenusServiceUpdateMenuRequestSuccessBlock)();
+typedef void(^MenusServiceMenusRequestSuccessBlock)(NSArray<Menu *> * _Nullable menus);
```

Comment:
It seems `_Nullable` is a bit more flexible, where-as `nullable` is more Obj-C(ish). So in this case with a block/function argument the complier wants `_Nullable` instead of `nullable`. But ultimately no difference as long as the specification is correct.

From Apple doc:

> You can use _Nullable and _Nonnull almost anywhere you can use the normal C const keyword, though of course they have to apply to a pointer type. However, in the common case there’s a much nicer way to write these annotations: within method declarations you can use the non-underscored forms nullable and nonnull immediately after an open parenthesis, as long as the type is a simple object or block pointer.

https://developer.apple.com/swift/blog/?id=25